pkgs/development/python-modules/cddlparser/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  # dependencies
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cddlparser";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-EcGxDRGFri0nS1GpoPXnijE7xBg2RuBle7U8Byiw2ns=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+  ];
+
+  meta = {
+    homepage = "https://github.com/tidoust/cddlparser";
+    downloadPage = "https://github.com/tidoust/cddlparser/releases";
+    description = "Concise data definition language (RFC 8610) parser implementation in Python.";
+    longDescription = ''
+      A CDDL parser in Python
+
+      Concise data definition language (RFC 8610) parser implementation in Python.
+    '';
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
We try to avoid as much as possible to add new packages without maintainers, because this make the package poorly maintained. Can you consider adding yourself as the maintainer?